.stylelintrc.json

```diff
@@ -29,7 +29,7 @@
     "unit-no-unknown": [true],
     "alpha-value-notation": ["percentage"],
     "hue-degree-notation": ["number"],
-    "color-function-notation": ["modern"],
+    "color-function-notation": ["legacy"],
```

Comment:
I would be interested to hear if you encountered any practical issues with the `modern` notation for color values, we could take that perspective in consideration for our own linting rules.

Reply:
This was specifically to use the format you provided in your example. I believe @joostvanviegen received linter warnings when using percentages in the HSL notation.

Reply:
Oh you were quick then! I accidentally had used the old notation, and I had updated my example to use the `modern` instead of the `legacy` notation, but apparently not before you copied it and ran into issues. There are some advantages to using the new modern notation, so I would recommend sticking with the modern notation instead of using my old example.